content/docs/azure/authentication.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Restricting Access & Authentication "
+description = "Restrict access of your deployment to specified IP addresses"
+weight = 100
++++
+
+This section shows how to restrict access to only certain IP addresses for your LoadBalancer Service. At a later date, it will also include formal authentication through Azure. This method is not the most ideal way to secure your Kubernetes cluster, as it requires that you access the service from the same IP address every time. This process was adapted from [this guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service).
```

Comment:
Recommendation: Add the word "Azure" a bit earlier on the page, so people know this doc applies specifically to Kubeflow on Azure - either the description in the front matter, or the first sentence on the page, are good places. (Even though the technique described here can probably be used across other cloud providers too, it's good to remind readers that this is the way currently recommended for Azure but not necessarily for other clouds.)